src/notation/internal/notationinteraction.cpp

```diff
@@ -1711,8 +1711,12 @@ bool NotationInteraction::applyPaletteElement(mu::engraving::EngravingItem* elem
 
             // A text should only be added at the start of the selection
             // There shouldn't be a text at each element
-            for (staff_idx_t staff = firstStaffIndex; staff < lastStaffIndex; staff++) {
-                applyDropPaletteElement(score, firstSegment->firstElement(staff), element, modifiers);
+            if (element->isTempoText()) {
```

Comment:
This fixes the problem for tempo text only, but it's problematic for any system-related item, for example System Text or Rehearsal Marks. Better to use `element->systemFlag()` instead.